test/Extensions.Mcp.EndToEndTests/Abstractions/CoreToolsProjectBase.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Extensions.Mcp.EndToEndTests.AbstractionOverCoreTools;
+using Microsoft.Azure.Functions.Tests.E2ETests;
+using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace Extensions.Mcp.EndToEndTests.Abstractions;
+
+public abstract class CoreToolsProjectBase: IAsyncLifetime
```

Comment:
```suggestion
public abstract class CoreToolsProjectBase : IAsyncLifetime
```

Do we not have style configured on this repo?